apps/condo/domains/marketplace/components/MarketplacePaymentsContent.tsx

```diff
@@ -0,0 +1,242 @@
+import EmptyListView from '@condo/domains/common/components/EmptyListView'
+import { DEFAULT_PAGE_SIZE, Table } from '@condo/domains/common/components/Table'
+import { TableFiltersContainer } from '@condo/domains/common/components/TableFiltersContainer'
+import { DEFAULT_BORDER_RADIUS } from '@condo/domains/common/constants/style'
+import { useSearch } from '@condo/domains/common/hooks/useSearch'
+import { getPageIndexFromOffset, parseQuery } from '@condo/domains/common/utils/tables.utils'
+
+
+const ROW_GUTTERS: RowProps['gutter'] = [0, 40]
+const PaymentsSumContainer = styled.div`
+  max-width: calc(100% + 48px);
+  border-radius: ${DEFAULT_BORDER_RADIUS};
+  border-color: ${colors.gray};
+  padding: 16px;
+  background-color: ${colors.white};
+
+  &.disabled {
+    opacity: 0.5;
+    pointer-events: none;  
+  }
+`
+
+export const MarketplacePaymentsContent = ({
+    searchPaymentsQuery,
+    tableColumns,
+    sortBy,
+    filterMetas,
+    role,
+}) => {
+    const intl = useIntl()
+    const EmptyListLabel = intl.formatMessage({ id: 'pages.condo.marketplace.payments.empty.text' })
+    const EmptyListTitle = intl.formatMessage({ id: 'pages.condo.marketplace.payments.empty.title' })
+    const SearchPlaceholder = intl.formatMessage({ id: 'filters.FullSearch' })
+    const ClearListSelectedRowMessage = intl.formatMessage({ id: 'global.cancelSelection' })
+    const allPaymentsSumMessage = intl.formatMessage({ id: 'pages.condo.marketplace.payments.stats.allPayment' })
+    const donePaymentsSumMessage = intl.formatMessage({ id: 'pages.condo.marketplace.payments.stats.donePayment' })
+    const inProcessPaymentsSumMessage = intl.formatMessage({ id: 'pages.condo.marketplace.payments.stats.inProcessPayment' })
+
+    const router = useRouter()
+    const client = useApolloClient()
+
+    const { filters, offset } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+    const canReadPayments = get(role, 'canReadPayments', false)
+
+    const {
+        loading: paymentsLoading,
+        count: total,
+        objs: payments,
+        refetch,
+    } = Payment.useObjects({
+        sortBy,
+        where: searchPaymentsQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    }, {
+        fetchPolicy: 'network-only',
+    })
+
+    const [allPaymentsSum, setAllPaymentsSum] = useState(0)
+    const [donePaymentsSum, setDonePaymentsSum] = useState(0)
+    const [inProcessPaymentsSum, setInProcessPaymentsSum] = useState(0)
+
+    useDeepCompareEffect(() => {
+        const allPaymentsSumResponse = client.query({
+            query: SUM_PAYMENTS_QUERY,
+            variables: {
+                where: {
+                    ...searchPaymentsQuery,
+                },
+            },
+            fetchPolicy: 'network-only',
+        })
+        setAllPaymentsSum(get(allPaymentsSumResponse, 'data.sum', 0))
```

Comment:
`sum` will be in `data.result.sum`.

I suggest to rewrite it like this:
```suggestion
       client.query({
            query: SUM_PAYMENTS_QUERY,
            variables: {
                where: {
                    organization : { id: organization.id },
                    ...searchPaymentsQuery,
                },
            },
            fetchPolicy: 'network-only',
        })
            .then(allPaymentsSumResponse => {
                setAllPaymentsSum(get(allPaymentsSumResponse, 'data.result.sum', 0))
            })
```